### ISSUE TITLE
support naming the config file *.mjs like rollup.config.mjs

### ISSUE THREAD

**Issue by arlac77** [rollup/rollup#3189]
### Feature Use Case
renaming all ESM modules into *.mjs (even build time & cfg)

### Feature Proposal
support naming the rollup config file *rollup.config.mjs* (not *.js*)

currently an error is thrown
```sh
rollup -c tests/rollup.config.mjs 
[!] Error: Must use import to load ES Module: .../tests/rollup.config.mjs
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: ...tests/rollup.config.mjs
    at Object.Module._extensions..mjs (internal/modules/cjs/loader.js:1007:9)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:849:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at /usr/lib/node_modules/rollup/dist/bin/rollup:845:32
```

**Comment by shellscape** [comment]
@Rich-Harris thoughts?

**Comment by arlac77** [comment]
fixed with #3204
